usaspending_api/api_contracts/contracts/v2/search/spending_by_transaction_grouped.md

```diff
@@ -0,0 +1,409 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Spending By Transaction Grouped [/api/v2/search/spending_by_transaction_grouped/]
+
+## POST
+
+Searches for transaction records based on a provided set of filters and groups them by their prime award.
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object"
+            }
+
+    + Attributes (object)
+        + `filters` (required, AdvancedFilterObject)
+            These filters are applied at the Transaction level (`children`). They will not affect the Prime Awards (`results`) themselves
+        + `fields` (required, FieldNameObject)
+            The field names to include for `children` objects (Transactions) the response.
+        + `limit`: 5 (optional, number)
+            The number of `results` (Prime Awards) to include per page. The number of `children` (Transactions) objects will always be limited to 10
+            + Default: 10
+        + `page`: 1 (optional, number)
+            The page of `results` (Prime Awards) to return based on `limit`.
+            + Default: 1
+        + `sort`: `Prime Award ID` (required, string)
+            The field on which to order `results` objects (Prime Awards) in the response. The `children` (Transaction) objects will always be #TODO (determine transaction sort)
```

Comment:
was this todo left in intentionally?
is the default sort on the children also Prime Award ID?

Reply:
I think this sort order needs to be finalized by design, but I will go ahead and put a default of `Transaction Amount` to match our Spending by Transaction endpoint for now.